app/models/queue_item.rb

```diff
@@ -0,0 +1,24 @@
+class QueueItem < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :video
+
+  validates_presence_of :user
+  validates_uniqueness_of :position, scope: :user_id
+  validates_uniqueness_of :video_id, scope: :user_id
+
+  delegate :category, to: :video
+  delegate :title, to: :video, prefix: :video
+  delegate :title, to: :category, prefix: :category
+
+  def user_rating
+    r = user.reviews.find_by(video: self.video)
```

Comment:
You don't need to use `self.video` - https://github.com/bbatsov/ruby-style-guide#no-self-unless-required